src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -172,6 +174,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if (!$input->mustSuggestArgumentValuesFor('user-class')) {
+            return;
+        }
+
+        $suggestions->suggestValues($this->userClasses);
```

Comment:
```suggestion
        if (!$input->mustSuggestArgumentValuesFor('user-class')) {
            $suggestions->suggestValues($this->userClasses);

            return;
        }
```

While we usually favor early return in Symfony, let's not do that here (to reduce the diff and thus merge conflicts when adding new completion suggestions in the future).